wwwroot/init/act.json

```diff
@@ -11,7 +11,14 @@
             "name": "ACT Government",
             "type": "group",
             "isPromoted": true,
-            "isOpen": true
+            "isOpen": true,
+            "items": [ {
```

Comment:
This should be removed because it's not necessary and it's inconsistent with what's in `04_ACT_Government.json`.  The only purpose of this file is to make http://nationalmap.gov.au/#act put the ACT Government group on top and open it (i.e. set the isPromoted and isOpen properties).  You do need to make sure the `type` matches, though, so if you do change ACT Government to be the Socrata-querying group, don't forget to change the `type` in this file, too.